bundles/binding/org.openhab.binding.netatmo/src/main/java/org/openhab/binding/netatmo/internal/weather/NetatmoWeatherBinding.java

```diff
@@ -180,13 +180,19 @@ public void execute(OAuthCredentials oauthCredentials, Collection<NetatmoBinding
                                 }
                             }
                                 break;
+                            case BATTERYPERCENT:
+                            case BATTERYSTATUS:
                             case BATTERYVP:
                             case RFSTATUS:
                                 for (Device device : stationsDataResponse.getDevices()) {
                                     for (Module module : device.getModules()) {
                                         if (module.getId().equals(moduleId)) {
                                             switch (measureType) {
+                                                case BATTERYPERCENT:
                                                 case BATTERYVP:
```

Comment:
Does this mean that binding users can't retrieve the battery_vp value because it's returned as a percentage instead of its actual value?  Other the pr lgtm.